src/server.c

```diff
@@ -3670,10 +3670,6 @@ void call(client *c, int flags) {
         replicationFeedMonitors(c, server.monitors, c->db->id, argv, argc);
     }
 
-    /* Clear the original argv.
-     * If the client is blocked we will handle slowlog when it is unblocked. */
-    if (!c->flag.blocked) freeClientOriginalArgv(c);
-
```

Comment:
I am not sure we have an issue here, but you assume this will be cleaned when the client is reset, but what about all the multiple call calls in the multi/exec path?

Reply:
Good catch!